From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Lance <lance@tomato64>
Date: Mon, 16 Dec 2024 00:00:00 +0000
Subject: [PATCH] package/rockchip-rkbin: use OpenWrt-tested version for U-Boot 2024.10

OpenWrt uses commit a2a0b89b (2024-02-22) which provides binaries tested
with U-Boot 2024.10:
- BL31 v1.45
- TPL v1.16 (rk3588_ddr_lp4_2112MHz_lp5_2400MHz_v1.16.bin)

Buildroot's current version (f43a462e, 2025-01-24) provides TPL v1.18
which is untested with U-Boot 2024.10 on RK3588/RK3588S platforms.

This ensures compatibility for NanoPi R6S and other RK3588-based boards
using U-Boot 2024.10.

Signed-off-by: Lance <lance@tomato64>
---
 package/rockchip-rkbin/rockchip-rkbin.hash | 2 +-
 package/rockchip-rkbin/rockchip-rkbin.mk   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/package/rockchip-rkbin/rockchip-rkbin.hash b/package/rockchip-rkbin/rockchip-rkbin.hash
index 1111111111..2222222222 100644
--- a/package/rockchip-rkbin/rockchip-rkbin.hash
+++ b/package/rockchip-rkbin/rockchip-rkbin.hash
@@ -1,3 +1,3 @@
 # Locally computed
-sha256  4a70cdc877600b75c0a885a8ebe1aa68f0800a0010aa580c105cbd9386140fa7  rockchip-rkbin-f43a462e7a1429a9d407ae52b4745033034a6cf9-git4.tar.gz
+sha256  8918553eb7a4bd5e57aade89a71498da0b2cfc7546f56ab5dbb11501fb2b1f53  rockchip-rkbin-a2a0b89b6c8c612dca5ed9ed8a68db8a07f68bc0-git4.tar.gz
 sha256  d2765124a2c24d71aba3cbc6cdd6975a5a297821264eb6c26378a4c16cd16e92  LICENSE
diff --git a/package/rockchip-rkbin/rockchip-rkbin.mk b/package/rockchip-rkbin/rockchip-rkbin.mk
index 1111111111..2222222222 100644
--- a/package/rockchip-rkbin/rockchip-rkbin.mk
+++ b/package/rockchip-rkbin/rockchip-rkbin.mk
@@ -4,7 +4,7 @@
 #
 ################################################################################

-ROCKCHIP_RKBIN_VERSION = f43a462e7a1429a9d407ae52b4745033034a6cf9
+ROCKCHIP_RKBIN_VERSION = a2a0b89b6c8c612dca5ed9ed8a68db8a07f68bc0
 ROCKCHIP_RKBIN_SITE = https://github.com/rockchip-linux/rkbin.git
 ROCKCHIP_RKBIN_SITE_METHOD = git
 ROCKCHIP_RKBIN_LICENSE = PROPRIETARY
--
2.34.1
